doc/source/whatsnew/v1.1.0.rst

```diff
@@ -951,9 +951,12 @@ I/O
 - :func:`pandas.read_hdf` has a more explicit error message when loading an
   unsupported HDF file (:issue:`9539`)
 - Bug in :meth:`~DataFrame.read_feather` was raising an `ArrowIOError` when reading an s3 or http file path (:issue:`29055`)
+- Bug in :meth:`read_parquet` was raising a ``FileNotFoundError`` when passed an s3 directory path. (:issue:`26388`)
```

Comment:
I don’t think mean to include this line or the next

Reply:
In deed, I will remove it.